<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantenimientoPartidos</class>
 <widget class="QDialog" name="MantenimientoPartidos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>621</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Partidos</string>
   </property>
   <widget class="QLineEdit" name="letEquipo2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPosesion_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblEquipo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPosesion">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letEquipo1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPases_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblTEsquina">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiros de Esquina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letRemates_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTEsquina_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPAdelantada">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPases">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPAdelantada">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posicion Adelantada:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letFaltas">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letGoles">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTarjetasR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QRadioButton" name="rdbModificar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbEliminar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbInsertar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbSalir">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblGoles">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Goles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRemates">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remates:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTarjetasR_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letFaltas_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPosesion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Posesion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVS">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTEsquina">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTarjetasA_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPases">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pases:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letRemates">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblTarjetasA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tarjetas Amarrillas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFaltas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Faltas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letGoles_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letTarjetasA">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblTarjetasR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tarjetas Rojas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPAdelantada_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblFecha">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHora">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hora:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letFecha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letHora">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pboAceptar">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pboAceptar</sender>
   <signal>clicked()</signal>
   <receiver>MantenimientoPartidos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
